test/shared/stripAnsi.js

```diff
@@ -0,0 +1,11 @@
+var ansiPattern = [
+  '[\\u001B\\u009B][[\\]()#;?]*(?:(?:(?:[a-zA-Z\\d]*(?:;[a-zA-Z\\d]*)*)?\\u0007)',
+  '(?:(?:\\d{1,4}(?:;\\d{0,4})*)?[\\dA-PRZcf-ntqry=><~]))',
+].join('|');
+var ansiRegex = new RegExp(ansiPattern, 'g');
+
+function stripAnsi(value) {
```

Comment:
@tdeschryver I also think it's good and welcome to send a pr to `gulp-test-tools`.